src/game.cpp

```diff
@@ -9049,9 +9049,8 @@ bool game::walk_move( const tripoint &dest_loc )
             add_msg( m_warning, _( "You cannot pass obstacles whilst mounted." ) );
             return false;
         }
-        const double base_moves =
-            u.run_cost( mcost, diag ) * 100.0 / crit->get_speed();
-        const double encumb_moves = u.get_weight() * 1.0 / 480_gram;
+        const double base_moves = u.run_cost( mcost, diag ) * 100.0 / crit->get_speed();
+        const double encumb_moves = u.get_weight() / 4800_gram;
```

Comment:
This looks wrong.  The result of this division will be an `int`, so there's no need for `encumb_moves` to be a `double`.  Is that what you intended?